ENH: cleanup

yarikoptic opened the pull request:
- .m files without shebang, .pdf, etc are not executable scripts - removing exec bit
-  No need for OSX aux files in git

yarikoptic committed:
ENH: No need for OSX aux files in git

yarikoptic committed:
BF: .m files without shebang, .pdf, etc are not executable scripts - removing exec bit

richardedden commented:
Thanks a lot for your input! Could you flesh out your commit comments a
little more? If you can bear to use full words and sentences, I can figure
out better what you mean.

On Fri, Apr 19, 2019 at 4:32 PM Yaroslav Halchenko <notifications@github.com>
wrote:

>
>    - .m files without shebang, .pdf, etc are not executable scripts -
>    removing exec bit
>    - No need for OSX aux files in git
>
> ------------------------------
> You can view, comment on, or merge this pull request online at:
>
>   https://github.com/richardedden/Gannet3.0/pull/7
> Commit Summary
>
>    - ENH: No need for OSX aux files in git
>    - BF: .m files without shebang, .pdf, etc are not executable scripts -
>    removing exec bit
>
> File Changes
>
>    - *D* .DS_Store
>    <https://github.com/richardedden/Gannet3.0/pull/7/files#diff-0> (0)
>    - *M* .gitignore
>    <https://github.com/richardedden/Gannet3.0/pull/7/files#diff-1> (2)
>    - *M* AlignUsingCr.m
>    <https://github.com/richardedden/Gannet3.0/pull/7/files#diff-2> (0)
>    - *M* AlignUsingH2O.m
>    <https://github.com/richardedden/Gannet3.0/pull/7/files#diff-3> (0)
>    - *M* AlignUsingPeak.m
>    <https://github.com/richardedden/Gannet3.0/pull/7/files#diff-4> (0)
>    - *M* CoRegStandAlone/CoReg.m
>    <https://github.com/richardedden/Gannet3.0/pull/7/files#diff-5> (0)
>    - *M* CoRegStandAlone/CoRegStandAlone.m
>    <https://github.com/richardedden/Gannet3.0/pull/7/files#diff-6> (0)
>    - *M* FitChoCr.m
>    <https://github.com/richardedden/Gannet3.0/pull/7/files#diff-7> (0)
>    - *M* FitCr.m
>    <https://github.com/richardedden/Gannet3.0/pull/7/files#diff-8> (0)
>    - *M* FitPeaksByFrames2.m
>    <https://github.com/richardedden/Gannet3.0/pull/7/files#diff-9> (0)
>    - *M* FiveGaussModel.m
>    <https://github.com/richardedden/Gannet3.0/pull/7/files#diff-10> (0)
>    - *M* FourGaussModel.m
>    <https://github.com/richardedden/Gannet3.0/pull/7/files#diff-11> (0)
>    - *M* FreqPhaseShiftNest.m
>    <https://github.com/richardedden/Gannet3.0/pull/7/files#diff-12> (0)
>    - *M* GannetCoRegister.m
>    <https://github.com/richardedden/Gannet3.0/pull/7/files#diff-13> (0)
>    - *M* GannetDiscernDatatype.m
>    <https://github.com/richardedden/Gannet3.0/pull/7/files#diff-14> (0)
>    - *M* GannetLoad.m
>    <https://github.com/richardedden/Gannet3.0/pull/7/files#diff-15> (0)
>    - *M* GannetManual.pdf
>    <https://github.com/richardedden/Gannet3.0/pull/7/files#diff-16> (0)
>    - *M* GannetMask.m
>    <https://github.com/richardedden/Gannet3.0/pull/7/files#diff-17> (0)
>    - *M* GannetMask_GE.m
>    <https://github.com/richardedden/Gannet3.0/pull/7/files#diff-18> (0)
>    - *M* GannetMask_Philips.m
>    <https://github.com/richardedden/Gannet3.0/pull/7/files#diff-19> (0)
>    - *M* GannetMask_SiemensRDA.m
>    <https://github.com/richardedden/Gannet3.0/pull/7/files#diff-20> (0)
>    - *M* GannetMask_SiemensTWIX.m
>    <https://github.com/richardedden/Gannet3.0/pull/7/files#diff-21> (0)
>    - *M* GannetPreInitialise.m
>    <https://github.com/richardedden/Gannet3.0/pull/7/files#diff-22> (0)
>    - *M* GannetQuantify.m
>    <https://github.com/richardedden/Gannet3.0/pull/7/files#diff-23> (0)
>    - *M* GannetSegment.m
>    <https://github.com/richardedden/Gannet3.0/pull/7/files#diff-24> (0)
>    - *M* GaussModel.m
>    <https://github.com/richardedden/Gannet3.0/pull/7/files#diff-25> (0)
>    - *M* LorentzModel.m
>    <https://github.com/richardedden/Gannet3.0/pull/7/files#diff-26> (0)
>    - *M* PhilipsRead.m
>    <https://github.com/richardedden/Gannet3.0/pull/7/files#diff-27> (0)
>    - *M* SDATread.m
>    <https://github.com/richardedden/Gannet3.0/pull/7/files#diff-28> (0)
>    - *M* SDATreadMEGA.m
>    <https://github.com/richardedden/Gannet3.0/pull/7/files#diff-29> (0)
>    - *M* SiemensRead.m
>    <https://github.com/richardedden/Gannet3.0/pull/7/files#diff-30> (0)
>    - *M* SiemensTwixRead.m
>    <https://github.com/richardedden/Gannet3.0/pull/7/files#diff-31> (0)
>    - *M* TwoLorentzModel.m
>    <https://github.com/richardedden/Gannet3.0/pull/7/files#diff-32> (0)
>    - *M* VAXD_to_uint64le.m
>    <https://github.com/richardedden/Gannet3.0/pull/7/files#diff-33> (0)
>    - *M* VAXF_to_uint32le.m
>    <https://github.com/richardedden/Gannet3.0/pull/7/files#diff-34> (0)
>    - *M* VAXG_to_uint64le.m
>    <https://github.com/richardedden/Gannet3.0/pull/7/files#diff-35> (0)
>    - *M* freadVAXG.m
>    <https://github.com/richardedden/Gannet3.0/pull/7/files#diff-36> (0)
>    - *M* fwriteVAXD.m
>    <https://github.com/richardedden/Gannet3.0/pull/7/files#diff-37> (0)
>    - *M* image_center.m
>    <https://github.com/richardedden/Gannet3.0/pull/7/files#diff-38> (0)
>    - *M* loadRawKspace.m
>    <https://github.com/richardedden/Gannet3.0/pull/7/files#diff-39> (0)
>    - *M* precommit_unixtime
>    <https://github.com/richardedden/Gannet3.0/pull/7/files#diff-40> (0)
>    - *M* readraw_Gannet.m
>    <https://github.com/richardedden/Gannet3.0/pull/7/files#diff-41> (0)
>    - *M* uint32le_to_VAXF.m
>    <https://github.com/richardedden/Gannet3.0/pull/7/files#diff-42> (0)
>    - *M* uint64le_to_VAXG.m
>    <https://github.com/richardedden/Gannet3.0/pull/7/files#diff-43> (0)
>
> Patch Links:
>
>    - https://github.com/richardedden/Gannet3.0/pull/7.patch
>    - https://github.com/richardedden/Gannet3.0/pull/7.diff
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/richardedden/Gannet3.0/pull/7>, or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAJR2R22I4NFJHFZKAXCQYDPRITZ3ANCNFSM4HHHZ5QA>
> .
>

yarikoptic commented:
Something like 

- Removed OSX auxilary file (.DS_Store) and set it to be ignored  by git in the future
- Removed executable bit from .m files without shebang, .pdf and other files which are not scripts

? or please suggest alternative wording

richardedden commented:
OK - thanks.  I get the first one now - that seems reasonable.
I don't know what shebang is.
We definitely want the pdf manual to be included with the code when people
download it.
What other files did you delete?!?
Thanks,
Richard


On Mon, Apr 22, 2019 at 4:08 PM Yaroslav Halchenko <notifications@github.com>
wrote:

> Something like
>
>    - Removed OSX auxilary file (.DS_Store) and set it to be ignored by
>    git in the future
>    - Removed executable bit from .m files without shebang, .pdf and other
>    files which are not scripts
>
> ? or please suggest alternative wording
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/richardedden/Gannet3.0/pull/7#issuecomment-485534731>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAJR2R4NDBBDOI37SG7FHY3PRYLH5ANCNFSM4HHHZ5QA>
> .
>

yarikoptic commented:
https://en.m.wikipedia.org/wiki/Shebang_(Unix)

I didn't remove any file in the second commit - I only removed executable bit from the files which aren't intended to be executed
